src/server/index.jsx

```diff
@@ -134,6 +133,20 @@ const injectDefaultCacheHeader = (req, res, next) => {
   next();
 };
 
+const injectResourceHintsHeader = (req, res, next) => {
+  if (req.originalUrl.indexOf('/pidgin') === 0) {
+    const assetOrigins = getAssetOrigins('pidgin');
+    res.set(
+      'Link',
+      assetOrigins
+        .reduce((toReturn, domainName) => {
+          return `${toReturn}<${domainName}>; rel="dns-prefetch", <${domainName}>; rel="preconnect",`;
+        }, '')
+        .slice(0, -1),
```

Comment:
Would something like this work?:
```suggestion
        assetOrigins.map(
          domainName =>`<${domainName}>; rel="dns-prefetch", <${domainName}>; rel="preconnect"`,
        ).join(',')
```